tests/e2e/map/map_granules_imagery.spec.js

```diff
@@ -85,16 +85,17 @@ test.describe('Map: imagery and layer-picker interactions', () => {
           })
         })
 
-        await page.route(/colormaps\/TEMPO_L3_Cloud_Cloud_Fraction_Total/, async (route) => {
-          await route.fulfill({
-            json: {}
-          })
-        })
+        await page.route(/graphql$/, async (route) => {
+          const { query } = JSON.parse(route.request().postData())
 
-        await page.route(/colormaps\/TEMPO_L3_Cloud_Cloud_Pressure_Total/, async (route) => {
-          await route.fulfill({
-            json: {}
-          })
+          // Return empty colormap body
+          if (query.includes('GetColorMaps')) {
+            await route.fulfill({
+              data: {
+                colormaps: []
+              }
```

Comment:
Here you have an empty colormap response as data: { colormaps: [] }`, but on line 179 you have `json: {}`. they need to be the same